app/src/main/res/layout/activity_main.xml

```diff
@@ -28,7 +28,7 @@
                 android:background="?attr/colorPrimary"
                 android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
                 app:navigationIcon="@drawable/ic_action_menu"
-                app:navigationOnClickListener="@{viewModel.navigationIconClick}"
+                app:navigationOnClickListener="@{viewModel::navigationIconClick}"
```

Comment:
It's probably better to use lambdas for clicks e.g. `@{() -> viewModel.whatever()}`

Reply:
The benefit seems marginal. But can we do "@{() -> viewModel.drawerState.set(true)} directly in the xml? Then even I would see the benefit...